src/Runtime/Runtime/System.Windows.Controls/WORKINPROGRESS/MenuItem.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+#if MIGRATION
+namespace System.Windows.Controls
+#else
+namespace Windows.UI.Xaml.Controls
+#endif
+{
+    public partial class MenuItem : Button
+    {
+
+        //DependencyProperty defined in HeaderedItemsControl from which MenuItem inherits in Silverlight
+        public static readonly DependencyProperty HeaderTemplateProperty =
```

Comment:
Move this to `src/Runtime/Runtime/System.Windows.Controls/MenuItem.cs`
```suggestion
        [OpenSilver.NotImplemented]
        public static readonly DependencyProperty HeaderTemplateProperty =
```